tests/test_geocoding.py

```diff
@@ -1,3 +1,4 @@
+# coding: utf-8
```

Comment:
Strange, I didn't have any issues with encoding... I'm on a mac

Reply:
running poetry run pytest resolved this issue